database/migrations/2018_04_02_010612_add_barber_properties_to_users_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddBarberPropertiesToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
```

Comment:
What sort of users are you planning to have who aren't barbers?

Will you want their phone number, city, state, and/or image?